client/src/components/basic/input/BasicInput.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div class="d-flex input-field text-color primary-border primary-rounded">
-    <input class="input-text-box" placeholder=" "/>
+    <input class="input-text-box" :value="modelValue"
+    @input="$emit('update:modelValue', $event.target.value)"/>
```

Comment:
Можно использовать простол v-model, она должна связать обе стороны
https://vuejs.org/guide/essentials/forms

Reply:
VueCompilerError: v-model cannot be used on a prop, because local prop bindings are not writable.
Use a v-bind binding combined with a v-on listener that emits update:x event instead.
по этому сделано так, как сделано